spec/features/user_creates_orders_spec.rb

```diff
@@ -17,11 +17,11 @@
 
         user = User.create(username: "adam", password: "adam")
         d = Destination.create(continent: "North America", description: "North American continent description.",
-                            lat: 41.8369, long: -87.6847, image_url: "south-america.jpg" )
+                            image_url: "south-america.jpg" )
         a = Activity.create(name: "Hiking", description: "We loven to do ze hikes.", image_url: "hiker.jpg")
         Trip.create(destination_id: d.id, activity_id: a.id, trip_cost: 10000.00, name: "Canadian Rockies")
         dd = Destination.create(continent: "South America", description: "South American continent description.",
-                            lat: -42.8369, long: -17.6847, image_url: "south-america.jpg" )
+                            image_url: "south-america.jpg" )
```

Comment:
Align the elements of a hash literal if they span more than one line.<br>Space inside parentheses detected.